src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -10,9 +10,26 @@ Feature: Allergies Management
     And a user clicks No Known Allergy button
     Then the system add no known allergies into the allergies table
     And close browser
-
+  
   @run
   Scenario: Remove No known allergies
     And a user clicks Remove No Known Allergy icon
     Then the system displays unknown in the allergies table
+    And close browser
```

Comment:
no more need of closing/quitting the browser this way

Reply:
Alright